transmission-with-openvpn.json

```
@@ -72,15 +72,20 @@
                         "label": "LOCAL_NETWORK",
                         "index": 6
                     },
+                    "TRANSMISSION_WEB_UI": {
+                        "description": "Alternative web UI to use; Use 'default' to leave the default Transmission UI, other options are: combustion, kettu, transmission-web-control, flood-for-transmission, shift or transmissionic.",
```

```suggestion
                        "description": "Alternative web UI to use; Use 'default' to leave the default Transmission UI, other options are (enter in lowercase): combustion, kettu, transmission-web-control, flood-for-transmission, shift or transmissionic.",
```
Since there is an emphasis on uppercase in the Provider variable, it might make sense to indicate that these should be lowercase (I did not test whether it matters, but since the values are listed like that in the documentation).

---

I believe the lowercase does matter, so this is a good point. I'll add the modification :+1:

---

For this portion you can just select the "Commit Suggestion" button up there, and it will automatically merge/push it as an update (you'll see it on your side and in this PR). For the other suggestions I had, I couldn't figure out how to do that.